site2/docs/io-mongo-sink.md

```diff
@@ -0,0 +1,51 @@
+---
+id: io-mongo-sink
+title: MongoDB sink connector
+sidebar_label: MongoDB sink connector
+---
+
+The MongoDB sink connector pulls messages from Pulsar topics 
+and persists the messages to collections.
+
+## Configuration
+
+The configuration of the MongoDB sink connector has the following properties.
+
+### Property
+
+| Name | Type|Required | Default | Description 
+|------|----------|----------|---------|-------------|
+| `mongoUri` | String| true| " " (empty string) | The MongoDB URI to which the connector connects. <br><br>For more information, see [connection string URI format](https://docs.mongodb.com/manual/reference/connection-string/). |
+| `database` | String| true| " " (empty string)| The database name to which the collection belongs. |
+| `collection` | String| true| " " (empty string)| The collection name to which the connector writes messages. |
+| `batchSize` | int|false|DEFAULT_BATCH_SIZE | The batch size of writing messages to collections. |
+| `batchTimeMs` |long|false|DEFAULT_BATCH_TIME_MS| The batch operation interval in milliseconds. |
```

Comment:
Using `100` instead of DEFAULT_BATCH_SIZE looks clearer
Using `1000` instead of DEFAULT_BATCH_TIME_MS.

Reply:
OK, I've updated